app/mailers/notification_mailer.rb

```diff
@@ -12,6 +12,13 @@ def notify_user_registration(user)
     mail(to: @user.email, subject: 'Welcome to Punchclock')
   end
 
+  def resend_user_registration(user)
+    @user = user
+    user.touch(:confirmed_at)
+    @token = user.send(:set_reset_password_token)
```

Comment:
Do you feel a transaction block is necessary here?

Reply:
Do we need the touch?

Reply:
I think so yeah, otherwise the reset password request fails because this field is empty.